src/main/java/com/dabsquared/gitlabjenkins/GitLabPushTrigger.java

```diff
@@ -516,7 +516,7 @@ public static GitLabPushTrigger getFromJob(Job<?, ?> job) {
         GitLabPushTrigger trigger = null;
         if (job instanceof ParameterizedJobMixIn.ParameterizedJob) {
             ParameterizedJobMixIn.ParameterizedJob p = (ParameterizedJobMixIn.ParameterizedJob) job;
-            for (Trigger t : p.getTriggers().values()) {
+            for (Object t : p.getTriggers().values()) {
```

Comment:
unrelated

Reply:
This is what I am talking about in my opening comment. The return type of `jenkins.model.ParameterizedJobMixIn.ParameterizedJob#getTriggers` has changed that's why I need to do this change if I update dependency to jenkins-core. Otherwise I get the following compile error:
```Error:(519, 52) java: incompatible types: java.lang.Object cannot be converted to hudson.triggers.Trigger```